### DISCUSSION TITLE
How do I set up ep_extra?

### DISCUSSION THREAD

**Discussion by jcrawl16** [valinet/ExplorerPatcher#1086] [General]
I'd like to use ep_extra to have Windows 7's alt-tab appearance. However, the readme calls for DLLs that I can't find. I have Windows 7's alt-tab DLL but I can't find the ep_extra DLL and the other ones mentioned in the readme.

**Comment by valinet** [comment]
Grab the latest artifacts (go to Actions, click on "Version: 22621.1344.53.1" or whatever the latest build is, scroll down, download "ep_bin_multi_4f3dab5_master" or something like that). In there, you will find `ep_extra.dll` and `ep_extra_valinet.win7alttab.dll`. Drop those and `AltTab.dll` from Windows 7 in your `C:\Windows` directory. Restart File Explorer. Windows will display the Windows 7 window switcher when pressing `alt`+`tab`.
